edkahara/iReporter
# Ft create v2 edit report endpoint #162423286

__What does this PR do__
Create an endpoint that allows a user to edit their report's location and comment.

__Description of tasks to be completed.__
Create an api endpoint that facilitates the editing of a report's location and status. Test the api endpoint.

__How should it be manually tested?__
Clone the project. Check out the ft-create-v2-edit-report-endpoint-#162423286 branch. Start the environment using `env\Scripts\activate` and `env.bat` to set up environment variables. Run flask run on the terminal and go to Postman. To perform the tests run nosetests.

__Relevant pivotal tracker stories.__
#162423286
